### PR TITLE
Automate Automaton enhancements

### PR DESCRIPTION
**Pull request by TikaelSol**

### CODE REVIEW

File: packs/data/feats.db/arcane-eye.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
             "value": null
         },
         "description": {
-            "value": "<p>Your eye has been magically enhanced to pierce darkness. You gain darkvision.</p>\n<p><strong>Enhancement</strong> Your eye can see @UUID[Compendium.pf2e.conditionitems.Invisible]{Invisible} creatures in brief spurts. You can cast see invisibility as an arcane innate spell once per hour.</p>"
+            "value": "<p>Your eye has been magically enhanced to pierce darkness. You gain darkvision.</p>\n<p><strong>Enhancement</strong> Your eye can see @UUID[Compendium.pf2e.conditionitems.Invisible]{Invisible} creatures in brief spurts. You can cast<em> @UUID[Compendium.pf2e.spells-srd.See Invisibility]{See Invisibility}</em> as an arcane innate spell once per hour.</p>"
```
Comment (stwlam) [inline packs/data/feats.db/arcane-eye.json:13]:
```suggestion
            "value": "<p>Your eye has been magically enhanced to pierce darkness. You gain darkvision.</p>\n<p><strong>Enhancement</strong> Your eye can see @UUID[Compendium.pf2e.conditionitems.Invisible]{Invisible} creatures in brief spurts. You can cast <em>@UUID[Compendium.pf2e.spells-srd.See Invisibility]{See Invisibility}</em> as an arcane innate spell once per hour.</p>"
```

File: packs/data/ancestries.db/automaton.json [hunk 2]
```diff
@@ -89,6 +89,16 @@
...
                 "value": "{item|flags.pf2e.rulesSelections.size}"
+            },
+            {
+                "key": "ActiveEffectLike",
+                "mode": "override",
+                "path": "flags.pf2e.automaton",
+                "priority": 10,
+                "value": {
+                    "GreaterEnhancements": [],
+                    "LesserEnhancements": []
```
Comment (stwlam) [inline packs/data/ancestries.db/automaton.json:100]:
```suggestion
                    "greaterEnhancements": [],
                    "lesserEnhancements": []
```

**Comment by TikaelSol** [comment]
image

Reevaluate is a bit noisy

File: src/module/migration/migrations/809-automaton-enhancements.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        return {
+            key: "ActiveEffectLike",
+            path: "flags.pf2e.automaton.enhancements",
+            mode: "override",
+            value: { greater: [], lesser: [] },
+        };
+    }
+
+    override async updateItem(source: ItemSourcePF2e): Promise<void> {
+        if (!source.system.slug) return;
```
Comment (stwlam) [inline src/module/migration/migrations/809-automaton-enhancements.ts:18]:
```suggestion
```

**Pull request merged**